internal/terraform/node_resource_abstract_instance.go

```diff
@@ -1218,6 +1218,8 @@ func (n *NodeAbstractResource) generateHCLStringAttributes(addr addrs.AbsResourc
 		configschema.FilterOr(configschema.FilterReadOnlyAttributes, configschema.FilterDeprecatedAttribute),
 		configschema.FilterDeprecatedBlock)
 
+	delete(filteredSchema.BlockTypes, "timeouts")
```

Comment:
Can we figure _why_ this block type panics (maybe because a case of NestingSingle is not covered)? `timeouts` may be special within the legacy SDK, but it is a valid part of a schema and can't be universally ignored.